site/components/map/StopsMap.tsx

```diff
@@ -245,6 +241,10 @@ const Map = ({
                     {showSelectAllText ? "Select all stops" : "Unselect all stops"}
                 </button>
             ) : null}
+            {selected.length === 100 ? (
+                <Warning text={`Stop selection capped at 100, ${selected.length} stops currently selected`} />
+            ) : null}
+            {warningMessage ? <Warning text={warningMessage} /> : null}
```

Comment:
We should probably implement the same warningMessageText logic to stops map. (Sorry I should have done that in my last ticket)

I'd recommend creating a mapUtils files and moving warningMessageText from ServicesMap.tsx to that utils file. Then replicating the warningMessage logic from ServiceMap.tsx in StopsMap.tsx

Reply:
done